eZ/Publish/API/Repository/SettingService.php

```diff
@@ -14,12 +14,27 @@
 
 interface SettingService
 {
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException If setting with the given group and identifier could not be found
+     */
     public function loadSetting(string $group, string $identifier): Setting;
 
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException If the current user user is not allowed to update a setting
+     */
```

Comment:
`updateSetting` method should also be declared as potentially throwing `NotFoundException` (when failing to update a setting that is for example already deleted), despite not being ever thrown in the actual implementation (yet?).

In the same manner, as discussed, in my opinion `InvalidArgumentException` should remain, despite never being thrown from implementation atm.